global-styles.scss

```diff
@@ -0,0 +1,129 @@
+@import './styles/mixins/mixins.scss';
+@import './styles/colors/colors.scss';
+@import './styles//variables/variables.scss';
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+p, a {
+    font-size: $font-size;
+}
+body {
+    font-family: 'Roboto', sans-serif;
+    min-height: $max-viewport-height;
+    color: $primary-color;
+}
+header {
+    background-color: white; 
+    color: $primary-color; 
+    h1 {
+      margin: 0;
+    }
+    nav {
+        @include flexbox-space-between;
+        flex-direction: row;
+        #logo{
+            width: 100px;
+            height: 40px;
+            object-fit: contain;
+        }
+        #profile-icon{
+            width: 50px;
+            height: 50px;
+            border-radius: 50%;
+            cursor: pointer;
+        }
+    }
+}
+
+button {
+    display: block;
+    border: none;
+    border-radius: 4px;
+    padding: 10px 20px;
+    cursor: pointer;
+    background-color: $primary-color;
+    color: $text-color;
+    box-shadow: 0 2px 4px $primary-color;
+    transition: background-color 0.3s ease;
+    :hover{
+      background-color: $primary-button-hover-color;
+    }
+}
+
+#error {
+    visibility: hidden;
```

Comment:
could you let me know if visibility is hidden could be part of error styling or could it be separate

Reply:
Error messages would be always hidden at all places and then would be made visible once the error happens. That is the reason I have added visibility hidden